src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,227 @@
-""" This is a dummy example """
+import requests
+import pandas as pd
+import time
+import os
+import matplotlib.pyplot as plt
 
-def main():
-    raise NotImplementedError
 
-if __name__ == "__main__":
-    main()
+# def main():
+#     raise NotImplementedError
+
+# if __name__ == "__main__":
+#     main()
+
+
+def connect_to_api(url, method="GET", headers=None, params=None, data=None):
```

Comment:
what's the point of adding `None` as default here when obviously the request won't work with it?